bigbluebutton-html5/imports/api/whiteboard-multi-user/server/handlers/modifyWhiteboardAccess.js

```diff
@@ -1,37 +0,0 @@
-import { check } from 'meteor/check';
```

Comment:
Both handlers do essentially the same thing. I don't know that the modify access message handler should be deleted though because it's going to need to be added back in 2.1 because there won't be a GetWhiteboardAccess message anymore.